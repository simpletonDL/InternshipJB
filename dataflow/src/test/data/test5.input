x = 1
if 1 {
    x = 2
}
y = x