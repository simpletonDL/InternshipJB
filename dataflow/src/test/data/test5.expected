Assignment y = x not used in line 5
